recipes/oatpp-libressl/all/test_package/oatpp-libressl-test.cpp

```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main() {
+    // std::cout << "OK" << std::endl;	
```

Comment:
You should include a minimal example

Reply:
Example of using code or some function to test? If the second one there is no simple test :(

Reply:
A simple creation + destruction of a context/configuration/.. object should  be sufficient.

Reply:
Thank you. I did it. There is simple object creation and destruction.